### PR TITLE
Exclude brexit terms from keyword search

### PR DESCRIPTION
**Pull request by leenagupte**
Trello: https://trello.com/c/kklgC5ov

## What's changed
Remove stop words from the keyword search made to rummager. This will return fewer, more relevant results.

N.B. If your keyword search only contains stop words, the search behaves as if you haven't searched by any keywords and returns all results. 

### Before
Screen Shot 2019-03-21 at 16 02 09

Screen Shot 2019-03-21 at 16 03 09

### After
Screen Shot 2019-03-21 at 16 02 17

Screen Shot 2019-03-21 at 16 03 16

### CODE REVIEW

File: spec/lib/search_query_builder_spec.rb [hunk 1]
```diff
@@ -172,6 +173,88 @@
...
+            'facets' => facets,
+            'filter' => filter,
+            'reject' => reject,
+            'default_order' => default_order,
+            'default_documents_per_page' => 10
+          }
+        }
+      }
+
+      it "should not include stopwords in search" do
```
Comment (sihugh) [inline spec/lib/search_query_builder_spec.rb]: Is it worth testing one of the contractions here too?  Like `shouldn't` or something else with an apostrophe

Comment (leenagupte) [reply spec/lib/search_query_builder_spec.rb]: @sihugh I've updated the test here: 7cbbf34a9bf4896f7aa2a6fdac29b8c9598c1919

File: app/lib/search_query_builder.rb [hunk 2]
```diff
@@ -203,4 +211,26 @@ def facet_params
...
+  def stopwords
+    %w(
+     a about above after again against all am an and any are arent as at
+     be because been before being below between both but by
+     cant cannot could couldnt
+     did didnt do does doesnt doing dont down during
+     each
+     few for from further
+     had hadnt has hasnt have havent having he hed hell hes her here heres hers herself him himself his how hows
+     i id ill im ive if in into is isnt it its its itself
```
Comment (vanitabarrett) [inline app/lib/search_query_builder.rb]: `its` appears twice here

File: app/lib/search_query_builder.rb [hunk 3]
```diff
@@ -203,4 +211,26 @@ def facet_params
...
+     lets
+     me more most mustnt my myself
+     no nor not of off on once only or other ought our ours ourselves out over own
+     same shant she shed shell shes should shouldnt so some such
+     than that thats the their theirs them themselves then there theres these they theyd theyll theyre theyve this those through to too
+     under until up
+     very
+     was wasnt we wed well were weve were werent what whats when whens where wheres which while who whos whom why whys with wont would wouldnt
```
Comment (vanitabarrett) [inline app/lib/search_query_builder.rb]: `were` appears twice here

Comment (leenagupte) [reply app/lib/search_query_builder.rb]: 🤦‍♀️ Copy and paste error. Will fix.

File: app/lib/search_query_builder.rb (hunk 2 above)
Comment (leenagupte) [reply app/lib/search_query_builder.rb]: 🤦‍♀️ Copy and paste error. Will fix.

Comment (leenagupte) [reply app/lib/search_query_builder.rb]: @vanitabarrett Fixed and fixed up

File: app/lib/search_query_builder.rb (hunk 3 above)
Comment (leenagupte) [reply app/lib/search_query_builder.rb]: @vanitabarrett Fixed and fixed up

**Pull request merged**